docs/source/pbft-source-documentation.rst

```diff
@@ -0,0 +1,38 @@
+*************************
+PBFT Source Documentation
+*************************
+
+.. raw:: html
+
+    <p class="large-text">
+        <a href="pbft_doc/sawtooth_pbft/index.html">
+            PBFT rustdocs
+        </a>
+    </p>
+
+
+This implementation of PBFT is written in the systems programming language
+`Rust <https://www.rust-lang.org/en-US/>`__. The implementation is heavily
+based on the `original PBFT paper
+<https://www.usenix.org/legacy/events/osdi99/full_papers/castro/castro_html/castro.html>`__,
+but draws some inspiration from the PBFT derivative `MinBFT
+<http://homepages.gsd.inesc-id.pt/~mpc/pubs/Veronese-Efficient%20Byzantine%20Fault%20Tolerance.pdf>`__,
+as well as the idea for `Istanbul Byzantine fault tolerance
+<https://github.com/ethereum/EIPs/issues/650>`__ from the Ethereum network.
+
+**Dependencies:**
+
```

Comment:
I suggest an intro sentence here:

     Dependencies are defined _{specified?}_ in ``Cargo.toml``.

Better yet, you could just have this sentence with a link to the file in master, rather than duplicating the contents here. (Because the contents might change.)

If the contents need to stay, there might be a way to auto-include the relevant lines from ``Cargo.toml``.  If you'd like to try this, ask me for more info. (I haven't done this yet, but I think it would work.)